app/models/order.rb

```diff
@@ -0,0 +1,51 @@
+class Order < ActiveRecord::Base
+  has_many :order_items
+
+  validates :status, presence: true
+  [:email, :mailing_address, :zip, :name_on_card, :last_four, :card_exp].each do |attribute|
+    validates attribute, presence: true, on: :update
```

Comment:
Instead of doing an .each here you can use the Rails built-in `validates_presence_of` instance method.
e.g.
```ruby
validates_presence_of :email, :mailing_address, :zip, :name_on_card, :last_four, :card_exp, on: :update
```